.github/workflows/e2e.yaml

```diff
@@ -83,3 +83,107 @@ jobs:
       - name: Get all operator logs
         run: kubectl logs deployment.apps/special-resource-controller-manager -n special-resource-operator
         if: ${{ always() }}
+
+  e2e:
+    name: Prebuilt kernel module
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+
+      - name: Save the kernel version
+        run: echo "KERNEL_VERSION=$(uname -r)" >> $GITHUB_ENV
+
+      - name: Build the kernel module
+        run: make KERNEL_SRC_DIR="/usr/src/linux-headers-${KERNEL_VERSION}"
+        working-directory: ci/sro-kmod
+
+      - name: Install curl
+        run: |
+          sudo apt -y update
+          sudo apt -y install curl
+
+      - name: Download and install minikube
+        run: |
+          curl -LO https://storage.googleapis.com/minikube/releases/latest/minikube_latest_amd64.deb
+          sudo dpkg -i ./minikube_latest_amd64.deb
```

Comment:
Can we use the [curl](https://github.com/marketplace/actions/github-action-for-curl) action here?